.github/workflows/docs.yml

```diff
@@ -46,7 +46,9 @@ jobs:
           python -m pip install -r ../docs/requirements.txt
       - name: Build docs
         working-directory: docs
-        run: make html
+        run: |
+          make nbconvert
```

Comment:
@eddyxu i think this is why the docs didn't have the notebook as sections

Reply:
got it . thanks 🙇